plugins/services/src/js/components/forms/HealthChecksFormSection.js

```diff
@@ -39,7 +71,9 @@ class HealthChecksFormSection extends Component {
             <FormGroup
               className="column-3"
               showError={Boolean(errors.gracePeriodSeconds)}>
-              <FieldLabel>Grace Period (s)</FieldLabel>
+              <FieldLabel helpText={gracePeriodHelpText}>
```

Comment:
Mm, OK. I'm probably missing some context, but I believed the advantage of composition was to avoid components with flexible interfaces (like the `HashMapList`), which tend to introduce bugs. 

`FieldLabel` is itself a convenience instead of using multiple divs, so introducing a parameter which optionally adds a tooltip doesn't seem evil to me.